semgrep/semgrep/constants.py

```diff
@@ -18,7 +18,7 @@
 YML_EXTENSIONS = {".yml", ".yaml"}
 
 try:
-    from semgrep.version import version  # type: ignore
+    from semgrep.version import version
```

Comment:
Mypy was complaining about this: `semgrep/semgrep/constants.py:21: error: unused 'type: ignore' comment`. Not sure what the issue is.

Reply:
Added this back in to fix CI, but it appears there's a mypy discrepancy between CI and local dev.